src/r4300/r4300_core.h

```diff
@@ -31,4 +37,30 @@ struct r4300_core
 
 void init_r4300(struct r4300_core* r4300);
 
+int64_t* r4300_regs(void);
+int64_t* r4300_mult_hi(void);
+int64_t* r4300_mult_lo(void);
+unsigned int* r4300_llbit(void);
+uint32_t* r4300_pc(void);
+
+uint32_t* r4300_last_addr(void);
+unsigned int* r4300_next_interrupt(void);
+
+unsigned int get_r4300_emumode(void);
```

Comment:
Why not `r4300_get_emumode()` to keep consistency in the function names? From what I've seen in the code every `r4300` related function start with `r4300`

Also this can be useful for modern IDE to find any `r4300` related function just typing them.